src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/LocalsTests.cs

```diff
@@ -2654,6 +2654,201 @@ void M()
             locals.Free();
         }
 
```

Comment:
Do these tests capture the case you described where "Transparent identifiers can be chained (i.e. have transparent
identifier properties)."?